### PR TITLE
refactor: 💡 remove route-action usage for workers

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12613

# Description

Remove route-action usage for workers. The only route-action usage that was not removed was the actions used for tag filtering because that will be replaced by worker tag ux improvements in the future.

## How to Test

1. start boundary dev instance
2. test editing and deleting an existing worker
3. start boundary server instance in order to get a registration token
4. use token to create a worker (make sure cancel and done buttons behave as intended in pki worker creation form)

## Checklist

- [ ] I have added before and after screenshots for UI changes
- [ ] I have added JSON response output for API changes
- [ ] I have added steps to reproduce and test for bug fixes in the description
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: ui/admin/app/routes/scopes/scope/workers.js [hunk 1]
```diff
@@ -6,9 +6,6 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
-import { loading } from 'ember-loading';
-import { confirm } from 'core/decorators/confirm';
-import { notifySuccess, notifyError } from 'core/decorators/notify';
 
 export default class ScopesScopeWorkersRoute extends Route {
   // =services
```
Comment (calcaide) [inline ui/admin/app/routes/scopes/scope/workers.js:11]: Do we need to keep service `router`?

**Review by calcaide — CHANGES_REQUESTED** [review]
I tested, successfully editing and deleting a worker.

Awesome job!!!

Do you mind providing the procedure or instructions to create a worker within the development environment?

**Review by calcaide — APPROVED** [review]
Amazing work @lisbet-alvarez 

Ignore my previous message 😅 as @DhariniJeeva suggested, I just follow the form instructions and got a worker up and running, my bad.

Thank you for this work 💪

**Review by cameronperera — APPROVED** [review]

File: ui/admin/app/routes/scopes/scope/workers.js (hunk 1 above)
Comment (lisbet-alvarez) [reply ui/admin/app/routes/scopes/scope/workers.js:11]: yup still needed on line 24

**Pull request merged**